### ISSUE TITLE
Feature Request; Moderation: Mute instance for X hours

### ISSUE THREAD

**Issue by GibsonGibson** [tootsuite/mastodon#14958]
I would propose the capability to mute an instance for a specified period of time for when the meta gets deep. This would allow a cooling off period without permanently defederating from the affected instances.

**Comment by SilverWolf32** [comment]
This sounds like a really good idea!

**Comment by joyeusenoelle** [comment]
I'd like to see timed silences for both users and instances on the admin side, and timed mutes for both users and instances on the user side. I suggest that the times offered be pre-defined rather than user/admin-defined: perhaps 1 hour, 3 hours, 6 hours, 12 hours, or 24 hours.

Here are two use cases:

* An instance has been overwhelmed by spambots, and the moderators are all away at the moment. As a fellow instance admin, you know the moderation team of that instance is generally very good at dealing with spam, and you trust them to handle it, but you don't want to see the spam coming over to your instance. You, as an admin, silence the remote instance for 12 hours to give their moderators time to work.

* A user you follow has decided to livetoot E3, but they're not tagging or CWing their posts and you really don't care about E3. You, as a user, mute the other user for 3 hours, hoping they'll be done livetooting by then; you can re-mute them again afterward if you need to.

This feature takes the onus of remembering to unsilence/unmute off the user/admin and onto the software, which is much more likely to remember to do it.

**Comment by GibsonGibson** [comment]
I agree with Noelle, wasn't trying to ask for all of these features, but this would be the optimal implementation of this feature.

**Comment by Boundarybreaker** [comment]
This honestly feels like a pretty mandatory feature, to be honest.

**Comment by splacat** [comment]
Sounds like a good idea to me.

**Comment by MatthewTheGlutton** [comment]
Timed mutes for accounts and instances, usable by both users and admins, would be perfect.

**Comment by drskrzyk** [comment]
I've mentioned this (the primary feature of instance level silence) and timed/cool off mutes/blocks before. It probably isn't an overstatement that this is something the fediverse is going to need to stay together in the long term.

**Comment by psydwannabe** [comment]
I would absolutely make use of this on my instance, regularly!
